testutils/src/mock.ts

```diff
@@ -271,6 +275,7 @@ export const SessionConnectionMock = jest.fn<
   const name = kernel?.name || options.model?.kernel?.name || DEFAULT_NAME;
   kernel = kernel || new KernelMock({ model: { name } });
   const model = {
+    id: UUID.uuid4(),
```

Comment:
Moved here from line 282. This way, we guarantee that `model` has `id`, but allow `options.model` to override it.

Reply:
:+1: